README.markdown

```diff
@@ -95,6 +95,62 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+**NOTE**: There is a bug currently, that causes FakeFS to fail with the following error:
+```
+TypeError:
+  superclass mismatch for class File
+```
+
+To prevent this, you'll need to require 'pp' *BEFORE* you require 'fakefs/spec_helpers' in your
+spec_helpers file. Like this:
+
+```ruby
+require 'pp'
+require 'fakefs/spec_helpers'
+
+describe "my spec" do
+  include FakeFS::SpecHelpers
+end
+```
+
+Mocking existing files
+----------------------
+
+If you want to load existing files like configuration files which sit with your gem in a
```

Comment:
What. How would it contained CLONED!? I'm not getting this one. :) 

Also, actually, I found that if I don't create the file, but clone my folder, it still complains that the file is not there. :/ But that must be something on my side then. :)